pom.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.georchestra</groupId>
+  <artifactId>geoserver-root</artifactId>
+  <version>2.27.1-georchestra</version>
+  <packaging>pom</packaging>
+  <name>GeoServer 2.x root module</name>
+  <properties>
+    <skipTests>true</skipTests>
+    <gs.version>2.27.1</gs.version>
+    <gt.version>33.1</gt.version>
+    <geofence.version>3.8.0</geofence.version>
+    <jetty.version>9.4.57.v20241219</jetty.version>
+    <marlin.version>0.9.4.8</marlin.version>
+    <jackson2.version>2.18.2</jackson2.version>
+    <packageDatadirScmVersion>master</packageDatadirScmVersion>
+    <server>generic</server>
+    <!-- overrides the versions provided by spring-boot in the geOrchestra root pom -->
+    <spring.version>5.3.39</spring.version>
+    <spring.security.version>5.8.16</spring.security.version>
+    <hibernate.version>5.6.15.Final</hibernate.version>
+<!--
+    <commons-beanutils.version>1.8.0</commons-beanutils.version>
+    <commons-digester.version>1.7</commons-digester.version>
+-->
+  </properties>
+  <modules>
+    <module>src/main</module>
+    <module>src/restconfig</module>
+    <module>src/gwc</module>
+    <module>src/web/core</module>
+    <!-- this ones are not provided onto the osgeo m2 repo -->
+    <module>src/extension/wmts-multi-dimensional</module>
+    <module>src/extension/ogcapi</module>
+    <module>webapp</module>
```

Comment:
I was first thinking of a `georchestra-webapp` module under `src/` but this way is also fine for me. It may be even better, as it clearly separates upstream from georchestra.